storage/src/main/java/tech/pegasys/teku/storage/server/kvstore/dataaccess/V4FinalizedKvStoreDao.java

```diff
@@ -205,6 +207,17 @@ public Map<String, Long> getColumnCounts() {
     return columnCounts;
   }
 
+  public Map<String, Long> getBlobsSidecarColumnCounts() {
+    final Map<String, Long> columnCounts = new LinkedHashMap<>();
+
+    schema.getColumnMap().entrySet().stream()
+        .filter(
+            stringKvStoreColumnEntry -> stringKvStoreColumnEntry.getKey().contains("BLOBS_SIDECAR"))
```

Comment:
I'd put exact column names to avoid hitting something unexpected in the future. Also `counts` is a little confusing. But not too much :)